src/pages/søknad/steg/uførevedtak/Uførevedtak.tsx

```diff
@@ -1,42 +1,67 @@
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, RawIntlProvider } from 'react-intl';
 import { useFormik } from 'formik';
 import Bunnknapper from '../../bunnknapper/Bunnknapper';
-import { Søknadsteg } from '../../types';
 import { JaNeiSpørsmål } from '~/components/FormElements';
 import { useAppSelector, useAppDispatch } from '~redux/Store';
 import søknadSlice from '~/features/søknad/søknadSlice';
 import messages from './uførevedtak-nb';
-import TextProvider, { Languages } from '~components/TextProvider';
 import sharedStyles from '../../steg-shared.module.less';
 import { Nullable } from '~lib/types';
+import { Feiloppsummering } from 'nav-frontend-skjema';
+import yup, { formikErrorsTilFeiloppsummering, formikErrorsHarFeil } from '~lib/validering';
+import { useHistory } from 'react-router-dom';
+import sharedI18n from '../steg-shared-i18n';
+import { useI18n } from '../../../../lib/hooks';
 
 interface FormData {
     harUførevedtak: Nullable<boolean>;
 }
 
-const Uførevedtak = () => {
+const schema = yup.object<FormData>({
+    harUførevedtak: yup
+        .boolean()
+        .nullable()
```

Comment:
Hvorfor brukes nullable?

Reply:
For å få en penere feilmelding. Uten den får man
> harUførevedtak must be a `boolean` type, but the final value was: `null`. If "null" is intended as an empty value be sure to mark the schema as `.nullable()`

_Med_ `nullable` så får vi melding om at feltet er påkrevd.